tests/test_qube_coverage.py

```
@@ -0,0 +1,1922 @@
+
+class Test_Qube_Coverage(unittest.TestCase):
+
+    def runTest(self):
+
+        np.random.seed(98765)
+
+        ##################################################################################
+        # Test __init__ error cases
+        ##################################################################################
+        # Test example not a Qube
+        with self.assertRaises(TypeError):
```

Ruff (0.14.8)

Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)